carina-core/src/main/java/com/qaprosoft/carina/core/foundation/listeners/AbstractTestListener.java

```diff
@@ -128,7 +128,10 @@ public void onStart(ITestContext context) {
 
     @Override
     public void onTestStart(ITestResult result) {
+        // create new folder for test report
+        ReportContext.createNewTestDir(StringUtils.EMPTY);

```

Comment:
1. rename to createTestDir
2. implement 2 methods: one without args -> UUID for naming, and with actual parameter